openapi-codegen/core/src/main/scala/sttp/tapir/codegen/openapi/models/OpenapiModels.scala

```diff
@@ -29,7 +36,54 @@ object OpenapiModels {
       components: Option[OpenapiComponent],
       security: Seq[Map[String, Seq[String]]],
       pathsExtensions: Map[String, Json] = Map.empty
-  )
+  ) {
+    def resolveAllOfSchemas: OpenapiDocument = {
+      val resolvedComponents = components.map { cs =>
```

Comment:
This method is basically the entire implementation, and seems kinda too cheeky and basic to work, but nonetheless...